internal/module/services/version_manager/version_manager.go

```diff
@@ -104,6 +119,30 @@ func (s *VersionManagerService) BeginVersion(ctx context.Context, request *terra
 		return nil, CreateModuleVersionError
 	}
 
+	parsedVersion := versions.MustParseVersion(strings.ReplaceAll(request.Module.GetVersion(), "v", ""))
+
+	// Check if official version.
+	if parsedVersion.GreaterThan(versions.MustParseVersion("0.0.0")) {
+
+		// Create release service struct.
+		rs := &release.ReleaseService{
+			Db:     s.Db,
+			Table:  release.ReleaseTableName,
+			Schema: release.GetReleaseSchema(release.ReleaseTableName),
+		}
+
+		// Publish the release.
+		_, err = rs.Publish(ctx, &releasePkg.PublishRequest{
```

Comment:
Shouldn't we also pass organization, types, links, and description as input to Publish? Otherwise, we'd have only name and version of the release in database. Or is this handled in another place?  @adcharre 
```
// Publish the release.
		_, err = rs.Publish(ctx, &releasePkg.PublishRequest{
			Name:    request.Module.GetName(),
			Version: request.Module.GetVersion(),
		})
```

Reply:
Yes your correct, organisation is available for modules and type should be set to module.
We don't yet have links or description available from the module publishing to add here so can be ignored for the moment